.github/workflows/known-FPs.csv

```diff
@@ -58,3 +58,4 @@ a62b37e0-45d3-48d9-a517-90c1a1b0186b;Eventlog Cleared;Computer: WIN-06FB45IHQ35
 87911521-7098-470b-a459-9a57fc80bdfd;Sysmon Configuration Updated;.*
 0eb46774-f1ab-4a74-8238-1155855f2263;Disable Windows Defender Functionalities Via Registry Keys;.*
 e9d4ab66-a532-4ef7-a502-66a9e4a34f5d;NTLMv1 Logon Between Client and Server;.*
+ccb5742c-c248-4982-8c5c-5571b9275ad3;Potential Suspicious Findstr.EXE Execution;.*
```

Comment:
I looked through the actions and didn't find a run with that FP. Do you have a link? I want to avoid .* if possible and it's not too much work

Reply:
```suggestion
ccb5742c-c248-4982-8c5c-5571b9275ad3;Potentially Suspicious Findstr.EXE Execution;.*
```
also just renamed the rule :see_no_evil:

Reply:
An example FP would `ParentCommandLine: C:\\Windows\\system32\\cmd.exe /c tasklist.exe | findstr /i \"\"httpd.exe\"\" 2\u003eNUL`

See https://github.com/SigmaHQ/sigma/pull/4343

It's best as a `*` as this rule will require internal tuning more than us tuning it imo

Reply:
Were there multiples? I am fine adding `httpd.exe` instead of `.*`. I don't wanna tune the rule. Just maintain a more strict FP filter for our goodlogs. The list has some value if you can see the difference between: this rule is "really bad with lots of FPs" or "with some FPs".